### DISCUSSION TITLE
Creating multiple documentation routes for a single dynamic path route

### DISCUSSION THREAD

**Discussion by motishaku** [fastapi/fastapi#3097] [Questions]
Hi,
I am creating an API which has a route using the path `/report/{report_name}`.
The route checks in a database and if there is no object with the report name specified returns a 404 error.
I would like to show in the docs every possible route, instead of a single "dynamic" route.
Is that something possible to do?
Thank you in advance for any help.

**Comment by tiangolo** [comment]
It would probably be better to ask in a GitHub issue, following the template that will guide you to structure the question in a way that makes it easier to understand your intention and need, to be able to help you better. And also providing a self-contained, [minimal, reproducible, example](https://stackoverflow.com/help/minimal-reproducible-example) that I can copy-paste to replicate it.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
